packages/drivers/routerlicious-driver/src/documentService.ts

```diff
@@ -103,19 +104,21 @@ export class DocumentService implements api.IDocumentService {
      * @returns returns the document delta stream service for routerlicious driver.
      */
     public async connectToDeltaStream(client: IClient): Promise<api.IDocumentDeltaConnection> {
+        const ordererToken = await this.tokenProvider.fetchOrdererToken();
         return R11sDocumentDeltaConnection.create(
             this.tenantId,
             this.documentId,
-            this.tokenProvider.token,
+            ordererToken.jwt,
             io,
             client,
             this.ordererUrl);
     }
 
     public async branch(): Promise<string> {
+        const ordererToken = await this.tokenProvider.fetchOrdererToken();
         let headers: { Authorization: string } | null = null;
         headers = {
-            Authorization: `Basic ${fromUtf8ToBase64(`${this.tenantId}:${this.tokenProvider.token}`)}`,
+            Authorization: `Basic ${fromUtf8ToBase64(`${this.tenantId}:${ordererToken.jwt}`)}`,
```

Comment:
consistency missing: in some other place you made it conditional to ordererToken  !== undefined.

Reply:
Good point. I think that was for back-compat. We should remove the undefined check and enforce token for all client calls.